nether/portal.lua

```diff
@@ -29,6 +29,27 @@ do
 	end
 end
 
+-- Nether aware mods will need to know if a player is in the nether.
+nether['is_player_in_nether'] = function (player)
+	local pname = player:get_player_name()
+	if players_in_nether[pname] == nil then
+		return false
+	else
+		return true
+	end
+end
+
+-- Nether aware mods may have other means of moving players between the Nether and Overworld, and if so, they should tell us about it so we can keep track of the player state.
+nether['external_nether_teleport'] = function (player)
```

Comment:
Yeah, I already changed all of the usages of this function to the object style references. I left the declaration as a table style reference because I thought it important to help others just learning Lua to understand that there are multiple ways to use the table object, and the table style declaration was relatively clean. I have no problem changing it if you prefer it that way.